src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -71,24 +71,29 @@ const char* Helix::ModeEnums[] = {"pitch-height", "pitch-turns", "height-turns",
 
 PROPERTY_SOURCE(PartDesign::Helix, PartDesign::ProfileBased)
 
+// we purposely use not FLT_MAX because this would not be computable
+const App::PropertyFloatConstraint::Constraints floatTurns = { Base::toDegrees<double>(Precision::Confusion()), INT_MAX, 1.0 };
```

Comment:
Why Base::toDegrees? floatTurns doesn't seem to restrict an angle.

Reply:
Damn it. Seems my force push with the correction did not went through, not it is correct.